app/views/search/results.html.erb

```diff
@@ -7,6 +7,22 @@
       <%= submit_tag 'Search', :class => "btn btn-success" %>
     </span>
   </div>
+  <br/>
+  Feedback type
+  <div class="field input-group">
+    <%= select_tag 'feedback', options_for_select([["Any", ""], ["true positive", 1], ["false positive", 2], ["true negative", 3], ["needs edit", 4]], params[:feedback]), :class => "form-control" %>
```

Comment:
Not quite what I meant. You want something like `options_from_collection_for_select(FeedbackType.all, 'id', 'short_code')` and use the `include_blank: true` option to `select_tag`. Then you can test for `params[:feedback].present?` in the controller.